src/main/kotlin/uk/gov/dluhc/printapi/config/ThreadPoolConfiguration.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dluhc.printapi.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor
+
+@Configuration
+class ThreadPoolConfiguration {
+
+    @Bean
+    fun zipStreamProducerTaskExecutor(
+        @Value("\${thread-pool.zip.core-size:5}") coreSize: Int,
```

Comment:
Minor - can we put these defaults in `application.yml` ? Then at least all the property definitions and their values are in one place (saves hunting all over the code base in 6 months looking where we set a given property! :D )